gizmo.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    print(f"Hello {name}, how are things in {country}?")
+
+def spell(word):
+    wordlength = 0
+    letters = []
+    spelling = ""
+    for letter in word:
+        wordlength += 1
+        letters.append(letter)
+    for i in range (wordlength):
```

Comment:
You don't need to make a list of characters. You can iterate directly over a string:

```python
for character in word:
```

Reply:
Thanks for the tip.